lib/active_admin/orm/active_record/comments/comment.rb

```diff
@@ -6,7 +6,7 @@ class Comment < ActiveRecord::Base
     belongs_to :resource, polymorphic: true
     belongs_to :author,   polymorphic: true
 
-    unless Rails::VERSION::MAJOR > 3 && !defined? ProtectedAttributes
+    unless !defined? ProtectedAttributes
```

Comment:
Yep, style change, but it reads much better so I'll add it! (I'll add the rule to `RuboCop` as well :wink:)